ArduSub/joystick.cpp

```diff
@@ -93,17 +94,29 @@ void Sub::transform_manual_control_to_rc_override(int16_t x, int16_t y, int16_t
 
     uint32_t tnow = AP_HAL::millis();
 
+    int16_t zTot = z + zTrim;
+    int16_t yTot = y + yTrim;
+    int16_t xTot = x + xTrim;
+
+    if (!controls_reset_since_input_hold) {
+        zTot = zTrim + 500; // 500 is neutral for throttle
+        yTot = yTrim;
+        xTot = xTrim;
+        // if all 3 axes return to neutral, than we're ready to accept input again
+        controls_reset_since_input_hold = (z == 500 && y == 0 && x == 0);
```

Comment:
exactly equal to zero on all channels is a bit restrictive. I think we should expand this to a +/-5% or so range.